notebooks/models/text-to-text/T5/master_processing.ipynb

```diff
@@ -0,0 +1,391 @@
+     "start_time": "2023-08-03T12:29:30.613779Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "Int64Index: 3488 entries, 0 to 595\n",
+      "Data columns (total 11 columns):\n",
+      " #   Column                 Non-Null Count  Dtype \n",
+      "---  ------                 --------------  ----- \n",
+      " 0   Unnamed: 0             3488 non-null   int64 \n",
+      " 1   id                     3488 non-null   int64 \n",
+      " 2   context                3488 non-null   object\n",
+      " 3   ocr                    3486 non-null   object\n",
+      " 4   img                    3488 non-null   object\n",
+      " 5   Name                   3488 non-null   object\n",
+      " 6   Topic                  1852 non-null   object\n",
+      " 7   Marked for processing  3482 non-null   object\n",
+      " 8   Question 1             2182 non-null   object\n",
+      " 9   Question 2             420 non-null    object\n",
+      " 10  Question 3             61 non-null     object\n",
+      "dtypes: int64(2), object(9)\n",
+      "memory usage: 327.0+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "merged_df.info()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:30.708249Z",
+     "start_time": "2023-08-03T12:29:30.623936Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/7j/mf5p4rjs4pv7s2ctxyh2kzyr0000gn/T/ipykernel_40225/269196921.py:3: FutureWarning: The default value of regex will change from True to False in a future version.\n",
+      "  dff1[text_field] = dff1[text_field].str.replace(r\"http\\S+\", \"\")\n",
+      "/var/folders/7j/mf5p4rjs4pv7s2ctxyh2kzyr0000gn/T/ipykernel_40225/269196921.py:5: FutureWarning: The default value of regex will change from True to False in a future version.\n",
+      "  dff1[text_field] = dff1[text_field].str.replace(r\"@\\S+\", \"\")\n",
+      "/var/folders/7j/mf5p4rjs4pv7s2ctxyh2kzyr0000gn/T/ipykernel_40225/269196921.py:6: FutureWarning: The default value of regex will change from True to False in a future version.\n",
+      "  dff1[text_field] = dff1[text_field].str.replace(r\"[^A-Za-z0-9(),!?@\\'\\`\\\"\\_\\n]\", \" \")\n"
+     ]
+    }
+   ],
+   "source": [
+    "#cleaning text\n",
+    "def standardize_text(dff1, text_field):\n",
+    "    dff1[text_field] = dff1[text_field].str.replace(r\"http\\S+\", \"\")\n",
+    "    dff1[text_field] = dff1[text_field].str.replace(r\"http\", \"\")\n",
+    "    dff1[text_field] = dff1[text_field].str.replace(r\"@\\S+\", \"\")\n",
+    "    dff1[text_field] = dff1[text_field].str.replace(r\"[^A-Za-z0-9(),!?@\\'\\`\\\"\\_\\n]\", \" \")\n",
```

Comment:
I looked it up, it is still common practice to remove this stuff, but for our dataset we can maybe argue in favor of keeping them since they might provide extra context given the nature of slides. So I'll remove this regex.

Reply:
Yes, So I removed the regex and fine-tuned again. Also let's keep the results from earlier as well just for comparison. And removing most regex didn't improve metric but surely improved quality of some questions so yes this definitely Is the right way to approach preprocessing